roles/ceph-mon/tasks/deploy_monitors.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: check if monitor initial keyring already exists
+  command: ceph auth get-key mon.
```

Comment:
```suggestion
  command: ceph --cluster {{ cluster }} auth get-key mon.
```

Reply:
@leseb is ceph-ansible still supporting custom cluster names? (why?)

Reply:
Yes, it is, for OSP and for existing clusters too.